package.json

```diff
@@ -23,8 +23,8 @@
     "design-tokens": "yarn workspace @espressive/design-tokens",
     "framer:publish": "dotenv npx framer-cli publish",
     "lint": "eslint ./",
+    "test": "jest -c ./.jest/config.js --detectOpenHandles",
```

Comment:
@lordpixel is it possible to put this `detectOpenHandles` into the config file instead? Usually these types of configurations can be done with the CLI or the configuration file. Since we have a lot in our file, it will probably make things easier for our future selves to see that configuration and not miss that it is happening here.

Reply:
We actually need to remove that one, it was useful when fixing unit tests but now it just adds overhead to tests.
`detectOpenHandles` was removed as it is not needed anymore.